VultisigApp/VultisigApp/Model/Vault+ProtoMappable.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Vault+ProtoMappable.swift
+//  VultisigApp
+//
+//  Created by Johnny Luo on 17/7/2024.
+//
+
+import Foundation
+import VultisigCommonData
+
+extension Vault: ProtoMappable {
+    convenience init(proto: VSVault) throws {
+        self.init(name: proto.name)
+        self.pubKeyECDSA = proto.publicKeyEcdsa
+        self.pubKeyEdDSA = proto.publicKeyEddsa
+        self.signers = proto.signers
+        self.createdAt = Date.now
+        self.hexChainCode = proto.hexChainCode
+        proto.keyShares.forEach { s in
+            self.keyshares.append(KeyShare(pubkey: s.publicKey, keyshare: s.keyshare))
+        }
+        self.localPartyID = proto.localPartyID
+        self.resharePrefix = proto.resharePrefix
+        self.order = 0
```

Comment:
is it safe always pass 0 in order?

Reply:
pretty much , because order doesn't make sense in a backup